### ISSUE TITLE
Add  Footer Section

### ISSUE THREAD

**Issue by Avik1320** [Riya14-dangra/RealEstate-Web#22]
There is no **Footer Section** present right now so I want to add a footer section in it and it will also **enhance the UI** also.

**Comment by Riya14-dangra** [comment]
@Avik1320 as I reopened the issue you can start working on it and give me the status of your work within one day

**Comment by Avik1320** [comment]
Can I use Bootstrap ?

**Comment by Riya14-dangra** [comment]
> Can I use Bootstrap ?

Yes you can

**Comment by Avik1320** [comment]
Sorry but when I am trying to push my files in GitHub there is a problem upcoming showing permission denied. what to do with it?

**Comment by Avik1320** [comment]
![image](https://user-images.githubusercontent.com/98302991/210491531-b64b811b-bee8-4248-b6d0-2595867db2de.png)
Can you help me out ?

**Comment by Riya14-dangra** [comment]
> ![image](https://user-images.githubusercontent.com/98302991/210491531-b64b811b-bee8-4248-b6d0-2595867db2de.png) Can you help me out ?

Yes sure .... Go to the fork repo then click on sync fork button then click update branch after this alll get updated then clone the repo and made a changes and then make a new pull request
